data/themes/darktable.css

```diff
@@ -174,6 +174,11 @@
   min-height: 0;
 }
 
+.monospace
+{
+  font-family: monospace;
```

Comment:
Please add that new CSS setting on block starting line 1418 please (/* Set some specific fonts */ part).
I would also suggest to add a more specific tag (like hist-tooltip for example).